docs/source/install.md

```diff
@@ -31,7 +31,7 @@ $ python -m pip install "featuretools[complete]"
 ```
 ```{tab} NLP Primitives
 ```console
-$ python -m pip install "featuretools[nlp_primitives]"
+$ python -m pip install "featuretools[nlp]"
```

Comment:
I'm curious why this changed

Reply:
I tried to make `update-checker` work but setuptools doesn't like the dash vs underscore situation. I figured since it's easier with less words, why not just make it simpler?